odor/src/main/scala/PostgresClient.scala

```diff
@@ -109,14 +110,24 @@ class PostgresClient(val pool: PostgresConnectionPool)(implicit ec: ExecutionCon
     command: Command[PARAMS],
     params: PARAMS = Void,
   ): Future[Unit] = async {
-    await(
-      await(connection)
+    val conn           = await(connection)
+    val startTimeNanos = nowNano()
+    val result = await(
+      conn
         .query(
           command.sql,
           command.encoder.encode(params).map(_.orNull).toJSArray,
         )
         .toFuture,
     )
+    val affectedRows = Try(result.rowCount.toInt).orElse(Try(result.rows.length)).getOrElse(0)
```

Comment:
Can move this line into the `if` :)

Why is this not sufficient?
```
    val affectedRows = result.rowCount.toIntOption.getOrElse(result.rows.length)
```

Reply:
toIntOption does not exist on Double. And it's still crashing even with `Try`. I don't know why...

Reply:
found something that seems to work.

Reply:
You are right, it just takes the int part of the double.

Reply:
But how could `toInt` fail on a double? Is it null?

Reply:
> But how could `toInt` fail on a double?

javascript